{
  "cve_id": "CVE-2018-16733",
  "github_data": {
    "repository": "ethereum/go-ethereum",
    "fix_commit": "106d196ec4a6451efedc60ab15957f231fa85639",
    "related_commits": [
      "106d196ec4a6451efedc60ab15957f231fa85639",
      "106d196ec4a6451efedc60ab15957f231fa85639"
    ],
    "patch_url": null
  },
  "vulnerability_details": {
    "cvss_score": null,
    "cvss_vector": null,
    "cwe_id": "CWE-20",
    "description": "In Go Ethereum (aka geth) before 1.8.14, TraceChain in eth/api_tracer.go does not verify that the end block is after the start block.",
    "attack_vector": null,
    "attack_complexity": null
  },
  "temporal_data": {
    "published_date": "2018-09-08T15:29:00.877",
    "last_modified": "2024-11-21T03:53:15.490",
    "fix_date": null
  },
  "references": [
    {
      "url": "https://github.com/ethereum/go-ethereum/commit/106d196ec4a6451efedc60ab15957f231fa85639",
      "source": "cve@mitre.org",
      "tags": [
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://github.com/ethereum/go-ethereum/commit/106d196ec4a6451efedc60ab15957f231fa85639",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Third Party Advisory"
      ]
    }
  ],
  "collection_metadata": {
    "collected_at": "2025-01-11T22:59:35.804460",
    "processing_status": "raw"
  }
}